src/inmanta/data/model.py

```diff
@@ -338,3 +338,16 @@ def __init__(
         self.end = end
         self.first_id = first_id
         self.last_id = last_id
+
+
+class ResourceDetails(BaseModel):
+    resource_id: ResourceIdStr
+    resource_type: ResourceType
+    agent: str
+    attribute: str
+    resource_id_value: str
+    last_deploy: datetime.datetime
```

Comment:
This could do with some documentation. 

What does it mean: last successful deploy, last completed deploy, last started deploy?

Reply:
In fact, all of these attributes could do with some documentation

Reply:
If I understand correctly, where it's set, then it's the time of the last completed deploy (renamed the field and added docstring).